backend/app/main.py

```diff
@@ -41,7 +42,7 @@ class Sentence(BaseModel):
 
 # Initialize DynamoDB client
 dynamodb = boto3.resource('dynamodb', region_name='us-west-2', aws_access_key_id=config.AWS_ACCESS_KEY, aws_secret_access_key=config.AWS_SECRET_KEY)
-table = dynamodb.Table('Sentences')
+table = dynamodb.Table('Users')
```

Comment:
put the sentences table back, it's needed for other code, rename the new table to usersTable